File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/test/java/simulacionTaller/factories/PlaterFactoryTest.java
package simulacionTaller.factories;

import org.junit.jupiter.api.Test;
import simulacionTaller.models.Plater;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class PlaterFactoryTest {

    @Test
    void createTest() {
        Plater plater = PlaterFactory.create();
        // No nulo, para ver si se han asignado correctamente los datos
        assertNotNull(plater);
    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/test/kotlin/simulacionTaller/factories/PlaterFactoryTest.kt
package simulacionTaller.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class PlaterFactoryTest {
    @Test
    fun createTest() {
        val plater = PlaterFactory.create()
        // No nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(plater)
    }
}